openhands/integrations/github/github_service.py

```diff
@@ -79,991 +70,6 @@ def __init__(
     def provider(self) -> str:
         return ProviderType.GITHUB.value
 
-    async def _get_github_headers(self) -> dict:
```

Comment:
OK looked at the red diffs in this file make me understand better how this is a nice incremental change from where we're at.